course3/homework-string/my_string.cc

```diff
@@ -0,0 +1,64 @@
+#include <iostream>
+#include <cstring>
+#include "my_string.h"
+
+MyString::MyString(){
+	buffer = std::unique_ptr<char[]>(new char[1]);
+	buffer = '\0';
```

Comment:
I am trying to assign the null-termination character to the default MyString character.

I did this because if I didn't, valgrind complained about the default constructor and the size() method: Conditional jump or move depends on uninitialised value(s)